gitpod-io/gitpod
# [blobserve] Split blobserve into a component in its own right

This PR turns blobserve into a component in its own right. Until now it was woven into registry-facade. With those two components drifting apart w.r.t. to their deployment mode, it makes sense to split them up.

Fixes #2571 

### How to test
1. Ensure all service pods are starting just fine
2. Open a workspace and watch it start - it should start without an issue
3. Check ws-proxy logs, you should not see an issue.
4. Enable feature preview and start a workspace with registry-facade

- [x] /werft https=true
